src/components/articles/ArticleWithDetails.jsx

```diff
@@ -1,36 +1,76 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { FacebookShareButton, TwitterShareButton, EmailShareButton } from 'react-share';
+import { faFacebookSquare, faTwitterSquare } from '@fortawesome/free-brands-svg-icons';
+import { faEnvelope } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import TagList from './TagList';
 import ArticleFooter from './ArticleFooter';
 import Comments from './comments/Comments';
 
 const Article = ({ article, user, addComment }) => {
   const { title, body } = article;
+  const url = `https://ah-frontend-odin-staging.herokuapp.com/articles/${article.slug}`;
```

Comment:
The URL will change the minute the app is deployed from staging to production.. In  future, you could use the `API_URL` constant

Reply:
the api_url constant applies for the backend